build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -286,6 +286,12 @@ fun configureTests() {
         configureSpock()
         configureFlakyTest()
 
+        distribution {
+            this as TestDistributionExtensionInternal
+            // Dogfooding TD against ge-td-dogfooding in order to test new features and benefit from bug fixes before they are released
+            server.set(uri("https://ge-td-dogfooding.grdev.net"))
```

Comment:
Do we need to put this inside `if (BuildEnvironment.isCiServer)` ?

Reply:
Ah, we want to enable this even for local running?

Reply:
No, I think it should also be configured for local builds where TD can be enabled via system property.